src/components/PatchAlbum.jsx

```diff
@@ -0,0 +1,142 @@
+/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useState, useEffect } from 'react';
+import { Deserializer } from 'jsonapi-serializer';
+import * as Yup from 'yup';
+import { Formik, Form, Field } from 'formik';
+import { useNavigate, useParams } from 'react-router-dom';
+import useAuth from '../hooks/useAuth';
+
+export default function PatchAlbum() {
+  const { id } = useParams();
+  const [loading, setLoading] = useState(false);
+  const [message, setMessage] = useState('');
+  const [error, setError] = useState(false);
+  const [artists, setArtists] = useState([]);
+  const [album, setAlbum] = useState(null);
+  const { currentUser } = useAuth();
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    setLoading(true);
+    fetch(`${process.env.REACT_APP_API_URL}/api/artists`)
+      .then((response) => {
+        if (!response.ok) {
+          setError(true);
+          return response.text().then((msg) => Promise.reject(new Error(msg)));
+        }
+        return response.json();
+      })
+      .then((data) => new Deserializer({ keyForAttribute: 'camelCase' }).deserialize(data, (_error, artistList) => setArtists(artistList)))
+      .catch(() => setError(true));
+    fetch(`${process.env.REACT_APP_API_URL}/api/albums/${id}`)
+      .then((response) => {
+        if (!response.ok) {
+          setError(true);
+          return response.text().then((msg) => Promise.reject(new Error(msg)));
+        }
+        return response.json();
+      })
+      .then((data) => new Deserializer({ keyForAttribute: 'camelCase' }).deserialize(data, (_error, albumInfo) => setAlbum(albumInfo)))
+      .catch(() => setError(true))
+      .finally(() => setLoading(false));
+  }, []);
+
+  if (loading) {
+    return <p>Loading...</p>;
+  }
+
+  if (error) {
+    return <p>{error}</p>;
+  }
+  return (
+    <div className="form">
+      <Formik
+        initialValues={{
+          artistId: album?.artistId,
+          name: album?.name,
+          publishedAt: album?.publishedAt,
+          cover: album?.cover,
+        }}
+        validationSchema={Yup.object({
+          name: Yup.string()
+            .required('This field is required'),
+          publishedAt: Yup.date()
+            .required('This field is required'),
+          cover: Yup.string(),
+        })}
+        onSubmit={async (values) => {
+          setLoading(true);
+          const formData = new FormData();
+          Object.keys(values).forEach((key) => {
+            formData.append(key, values[key]);
+          });
+          const requestOptions = {
+            method: 'PATCH',
+            headers: {
+              Authorization: `Bearer ${currentUser?.access_token}`,
+            },
+            body: formData,
+          };
+          try {
+            const response = await fetch(`${process.env.REACT_APP_API_URL}/api/albums/${id}`, requestOptions);
+            if (response.status !== 201) {
```

Comment:
Este endpoint retorna `200`, por lo que debería ser:
```suggestion
            if (response.status !== 200) {
```